apps/www/components/layout/navbar.tsx

```diff
@@ -39,18 +39,7 @@ export function NavBar({
         <div className="flex items-center space-x-3">
           {rightElements}
 
-          {user ? (
-            <UserAccountNav user={user} />
-          ) : (
-            <Button
-              className="px-3"
-              variant="default"
-              size="sm"
-              onClick={signInModal.onOpen}
-            >
-              Sign In
-            </Button>
-          )}
+          <UserAccountNav />
```

Comment:
Is this going to be the same Sign In button that we have on the top right now? With the dropdown when looged inn? 

Can see that there is some dropdown bellow so i belive it is?

Reply:
Same button but with diffrent dropdown menu content "Clerk's prebuilt one". so Switching back to the old way.